runner.go

```diff
@@ -16,6 +16,7 @@ var (
 	configFile     string
 	nonInteractive bool
 	listTasks      bool
+	interactive bool
```

Comment:
naming is weird

usually i would expect this kind of flag to be called `--watch`

what is nonInteractive

Reply:
I would also expect it to be called `--watch` but since the other one is called nonInteractive I called it interactive.

Reply:
yeah but like what if you pass both interactive *and* noninteractive?

I don't think these two distinct axes should have the same name

Reply:
oh nm edwin hit this below :+1:

Reply:
Gonna just change this to just a watch flag then.